backend/utils/threads_utils.py

```diff
@@ -28,6 +28,39 @@ async def get_message_threads(org_id, room_id, message_id):
     message = await get_message(org_id, room_id, message_id)
     return message["threads"]
 
+async def get_message_thread(org_id, room_id, message_id, thread_id):
+    """Retrives a single the messages in a thread.
+
```

Comment:
Did you mean "Retrieves a single thread message"?

Reply:
yes, my bad. I've corrected it.